tools/deployment/package/docker-compose-all.yaml

```diff
@@ -470,8 +472,8 @@ services:
     environment:
       CLP_LOGGING_LEVEL: "${CLP_MCP_LOGGING_LEVEL:-INFO}"
       CLP_LOGS_DIR: "/var/log/mcp_server"
-      CLP_DB_USER: "${CLP_DB_USER}"
-      CLP_DB_PASS: "${CLP_DB_PASS}"
+      CLP_DB_PASS: "${CLP_DB_PASS:?Please set a value.}"
+      CLP_DB_USER: "${CLP_DB_USER:?Please set a value.}"
```

Comment:
unrelated to the current PR - `CLP_DB_PASS` should be an optional env as well (default to "clp-user")

@coderabbitai please create an issue to track